definitions/scenarios/upgrade_to_katello_3_14.rb

```diff
@@ -0,0 +1,86 @@
+module Scenarios::Katello_3_14
+  class Abstract < ForemanMaintain::Scenario
+    def self.upgrade_metadata(&block)
+      metadata do
+        tags :upgrade_scenario
+        confine do
+          feature(:instance).upstream? && feature(:foreman_server) && \
```

Comment:
Instead of adding `feature(:instance).upstream?`, can we just add `!feature(:instance).downstream?`

Reply:
Yes, added `upstream?` for readability, however if it does not make sense then happy to use `!feature(:instance).downstream?`.

Reply:
could you please remove `upstream?` method.